### PR TITLE
BUGFIX #906

### PR DESCRIPTION
**Pull request by v-akarke**

### CODE REVIEW

**Comment by SteveMutungi254** [comment]
@v-akarke , @v-varshamane : Let's remove the s at the end to name (`Find-EntraPermission`) to comply with PowerShell best practice of singular nouns.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ be74164b-cff1-491c-8741-e671cb536e13 Admin   Policy.ReadWrite.ApplicationConfigu
 
 ###Example 2. Get a list of permissions for the Read permissions of the Application domain.
 ```powershell
-PS C:\>Find-EntraPermissions application.Read | Format-List
+PS C:\>Find-EntraPermission application.Read | Format-List
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md:88]:
```suggestion
Find-EntraPermission application.Read | Format-List
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ supply input to commands
 ### Example 1: Get a list of all Application permissions.
 
 ```powershell
-PS C:\> Find-EntraPermissions application
+PS C:\> Find-EntraPermission application
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md:66]:
```suggestion
Find-EntraPermission application
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md [hunk 3]
```diff
@@ -105,16 +90,53 @@ Name           : Application.Read.All
...
+```powershell
+Find-EntraPermission -SearchString 'User.Read.All' -ExactMatch
+```
 
+```Output
+   PermissionType: Delegated
+
+Id                                   Consent Name          Description
+--                                   ------- ----          -----------
+a154be20-db9c-4678-8ab7-66f6cc099a59 Admin   User.Read.All Allows the app to read the full set of profile properties, reports, and ma…
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md:104]: Use fake GUIDs in this md file.

Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md:104]: @KenitoInc: Permission strings and Role Template IDs are exempted as they are public info.

image

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**